ldes-server-port-ingest-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/rest/validators/memberingestvalidator/MemberIngestValidator.java

```diff
@@ -45,10 +47,22 @@ private void validateModel(Model model, EventStream eventStream) {
         if (memberSubjects.size() > 1 && !eventStream.isVersionCreationEnabled()) {
             throw new IngestValidationException("Only 1 member is allowed per ingest");
         }
-        int timestampPathCount = model.listStatements(null, ResourceFactory.createProperty(eventStream.getTimestampPath()), (RDFNode) null).filterKeep(statement -> memberSubjects.contains(statement.getSubject())).toList().size();
-        int versionOfPathCount = model.listStatements(null, ResourceFactory.createProperty(eventStream.getVersionOfPath()), (RDFNode) null).filterKeep(statement -> memberSubjects.contains(statement.getSubject())).toList().size();
-        boolean conformsTimePath = eventStream.isVersionCreationEnabled() ? timestampPathCount == 0 : timestampPathCount == 1;
-        boolean conformsVersionPath = eventStream.isVersionCreationEnabled() ? versionOfPathCount == 0 : versionOfPathCount == 1;
+        List<Statement> timestampStatements = model.listStatements(null, ResourceFactory.createProperty(eventStream.getTimestampPath()), (RDFNode) null).filterKeep(statement -> memberSubjects.contains(statement.getSubject())).toList();
+        List<Statement> versionOfStatements = model.listStatements(null, ResourceFactory.createProperty(eventStream.getVersionOfPath()), (RDFNode) null).filterKeep(statement -> memberSubjects.contains(statement.getSubject())).toList();
+
+        timestampStatements.forEach(statement -> {
+            if (!statement.getObject().isLiteral() || !Objects.equals(statement.getObject().asLiteral().getDatatype().getURI(), XML_DATETIME.getURI())) {
```

Comment:
Instead of comparing the URI's, you could compare the `[..].asLiteral().getDataType()` to `XSDDatatype.XSDdateTime`, which is built in in Jena , resulting in less maintaining the RdfConstants